x/gov/client/testutil/cli_test.go

```diff
@@ -21,7 +21,7 @@ func TestIntegrationTestSuite(t *testing.T) {
 	suite.Run(t, NewIntegrationTestSuite(cfg))
 
 	genesisState := types.DefaultGenesisState()
-	genesisState.DepositParams = types.NewDepositParams(sdk.NewCoins(sdk.NewCoin(cfg.BondDenom, types.DefaultMinDepositTokens)), time.Duration(15)*time.Second, sdk.NewCoins(sdk.NewCoin(cfg.BondDenom, types.DefaultMinExpeditedDepositTokens)))
+	genesisState.DepositParams = types.NewDepositParams(sdk.NewCoins(sdk.NewCoin(cfg.BondDenom, types.DefaultMinDepositTokens)), time.Duration(15)*time.Second, sdk.NewCoins(sdk.NewCoin(cfg.BondDenom, types.DefaultMinExpeditedDepositTokens)), 0)
```

Comment:
Let's avoid changing function signatures on functions in the cosmos-sdk repo if possible.

To avoid situations like this:  https://github.com/osmosis-labs/cosmos-sdk/issues/279